app/AppDetail.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Suyabay;
+
+use Illuminate\Database\Eloquent\Model;
+
+class AppDetail extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name',
+        'user_id',
+        'homepage_url',
+        'description',
+        'api_token'
+    ];
+
+    public function user()
```

Comment:
**Code Quality - Nitpick**
@andela-araimi Please give this method some comment and do same for all other methods lacking comment.